python/ray/serve/api.py

```diff
@@ -498,11 +498,7 @@ def run(
     # Record after Ray has been started.
     record_extra_usage_tag(TagKey.SERVE_API_VERSION, "v2")
 
-    if isinstance(target, Application):
```

Comment:
Why don't we support `Application` anymore?

Reply:
Passing `Application` into `serve.run` was only used by the deploy task. Since we are removing that, I removed support for this - WDYT?